app/search_builders/hyrax/abstract_type_relation.rb

```diff
@@ -14,7 +14,7 @@ def equivalent_class?(klass)
 
     def search_model_clause
       clauses = allowable_types.map do |k|
-        ActiveFedora::SolrQueryBuilder.construct_query_for_rel(has_model: k.to_s)
+        ActiveFedora::SolrQueryBuilder.construct_query_for_rel(Valkyrie::Persistence::Solr::Queries::MODEL => k.to_s)
```

Comment:
I don't think this is right since `construct_query_for_rel` is going to run `solr_name` and I think end up with "internal_resource_ssim_ssim".

Reply:
Oops, looks like you're right. I thought it behaved differently for string keys.